FreeTAKServer/controllers/DatabaseControllers/ActiveEmergencysController.py

```diff
@@ -6,18 +6,19 @@
 class ActiveEmergencysController(TableController):
     def __init__(self):
         self.table = ActiveEmergencys
-    
+
     def create(self, session, object):
         row = ActiveEmergencys()
         row.event = EventTableController().convert_model_to_row(object, row.event)
         session.add(row)
         session.commit()
         return row
 
+
 if __name__ == "__main__":
-    from FreeTAKServer.controllers.DatabaseControllers.DatabaseController import DatabaseController
+    # TODO: This is broken. SimulateSentCOT() does not exist.
```

Comment:
Self-explanatory comment added. This is broken. `SimulateSentCOT()` does not exist.

Reply:
yes this was for testing purposes will open an issue so it is removed